editor/editor_spin_slider.cpp

```diff
@@ -188,6 +188,67 @@ void EditorSpinSlider::_grabber_gui_input(const Ref<InputEvent> &p_event) {
 	}
 }
 
+void EditorSpinSlider::_value_input_gui_input(const Ref<InputEvent> &p_event) {
+	Ref<InputEventKey> k = p_event;
+	if (k.is_valid() && k->is_pressed()) {
+		double step = get_step();
+		double real_step = step;
+		if (step < 1) {
+			double divisor = 1.0 / get_step();
+
+			if (trunc(divisor) == divisor) {
+				step = 1.0;
+			}
+		}
+
+#ifdef APPLE_STYLE_KEYS
+		if (k->get_command()) {
+#else
+		if (k->get_control()) {
+#endif
+			step *= 100.0;
+		} else if (k->get_shift()) {
+			step *= 10.0;
+#ifdef APPLE_STYLE_KEYS
+		} else if (k->get_metakey()) {
+#else
+		} else if (k->get_alt()) {
+#endif
```

Comment:
I'm surprised by your findings on `get_alt()` vs `get_metakey()`, that doesn't seem to match other `APPLE_STYLE_KEYS` checks in the codebase.

But let's merge and see if any macOS user reports unexpected behavior.